Description:
  endian.h is found in libbsd-dev, but -I/usr/include/bsd will produce
  thousands of errors, so hardcode the location is better.
Author: Kai-Chung Yan <seamlikok@gmail.com>
Last-Update: 2015-08-05
--- a/include/libunwind_i.h
+++ b/include/libunwind_i.h
@@ -68,7 +68,7 @@
 #if defined(HAVE_ENDIAN_H)
 # include <endian.h>
 #elif defined(HAVE_SYS_ENDIAN_H)
-# include <sys/endian.h>
+# include <bsd/sys/endian.h>
 #else
 # define __LITTLE_ENDIAN	1234
 # define __BIG_ENDIAN		4321
